cclib/io/ccio.py

```diff
@@ -83,6 +84,7 @@
     (DALTON,    ["Dalton - An Electronic Structure Program"],       True),
     (FChk,      ["Number of atoms", "I"],                           True),
     (GAMESS,    ["GAMESS"],                                         False),
+    (GAMESSDAT, ["GAMESS .dat files"],                                         False),
```

Comment:
This trigger should be something like `$VEC` or something that appears in the .dat files